lib/language_pack/base.rb

```diff
@@ -35,10 +36,24 @@ def initialize(build_path, cache_path = nil, layer_dir=nil)
     @id            = Digest::SHA1.hexdigest("#{Time.now.to_f}-#{rand(1000000)}")[0..10]
     @fetchers      = {:buildpack => LanguagePack::Fetcher.new(VENDOR_URL) }
     @layer_dir     = layer_dir
+    @arch = get_arch(stack: @stack)
 
     Dir.chdir build_path
   end
 
+  def get_arch(stack:)
+    output = run!("arch", silent: true).strip
```

Comment:
If you use `dpkg --print-architecture` then you won't need the mapping, since `dpkg --print-architecture` prints `amd64` and `arm64`.